README.md

```diff
@@ -1,2 +1,88 @@
-# sb1-open-api-tutorial
-This application shows how one sets up a connection to the SpareBank1 API, getting an oauth token and retrieving  data from a sandbox or production environment.
+# SpareBank1 Open API Tutorial
+
+## Introduction
+This application shows how one sets up a connection to the SpareBank1 API, getting an oauth token and retrieving 
+data from a sandbox or production environment.
+ 
+Example in this application sends a request to get users accounts and prints them out. Then the application sends another 
+request to get all transaction for an account and prints out the newest.  
+
+## Build Application
+```` 
+mvn clean install
+````
+## Set up for sandbox environment  
+### 1. Register application
+Go to the sandbox environment [developersparebank1](https://developersparebank1.no/). Create a user and register your 
+application.
+
+### 2. Configure properties for sandbox
+Properties needed to run this application can be found [application.yaml](src/main/resources/application.yaml)
+
+    api_url:  https://developer-api.sparebank1.no
+    client_id: found where you registered you application
+    client_secret: found where you registered you application
+    redirect_url: the same redirect url that you registered with your application
+    authorization_code: Only used in production. Leave it blank for sandbox
+    account_id: running the application will print all accounts. Use the 'id' field from the response
+    
+## Set up for production environment
+### 1. Register Application
+Go to [Personlig Klient](https://developersparebank1.no/personlig-klient) and choose the bank that you are a customer of
```

Comment:
Her kan vi bare referere til link tenker jeg. F.eks
Follow the steps here to create your credentials for Personlig klient (eller lignende)

Reply:
Er ikke helt enig her. https://developersparebank1.no/personlig-klient har flere steg enn det som er nødvendig til å kjøre applikasjonen f.eks Steg 3 Issue an Oauth-Token. Å lage oauth token gjøres i applikasjonen. Håper bare at det ikke blir noe forvirring rundt det. 

Jeg kan jo legge til at du trenger bare å utføre steg 1 i linken (og kanskje steg 2).